python/dgl/function/message.py

```diff
@@ -46,20 +44,7 @@ def __init__(self, binary_op, lhs, rhs, lhs_field, rhs_field, out_field):
 
     def _invoke(self, graph, src_frame, dst_frame, edge_frame, out_size,
```

Comment:
The entire `_invoke` method could be removed.

Reply:
fixed